sources/packages/backend/apps/queue-consumers/src/queue-consumers.module.ts

```diff
@@ -71,6 +71,7 @@ import { MicroserviceHealthIndicator, TerminusModule } from "@nestjs/terminus";
 import { CASSupplierIntegrationService } from "./services/cas-supplier/cas-supplier.service";
 import { VirusScanProcessor } from "./processors/virus-scan/virus-scan.processor";
 import { CASService } from "@sims/integrations/cas/cas.service";
+import { ObjectStorageService } from "@sims/integrations/object-storage";
```

Comment:
Please import from the index.

Reply:
Didn't get it. It is pointing to index file.